harness/harness.go

```diff
@@ -104,7 +119,14 @@ func (c *H) flushToParent(format string, args ...interface{}) {
 
 	c.mu.Lock()
 	defer c.mu.Unlock()
-	io.Copy(p.w, &c.output)
+	b := c.output.Bytes()
```

Comment:
Technically we can save some memory by writing these two lines as:

```
outputBufferCopy := c.output
outputBufferCopy.WriteTo(p.w)
```

I think that works; It's okay if you don't want to change it from what you have.